lib/blockchain_double.js

```diff
@@ -478,9 +451,10 @@ BlockchainDouble.prototype.sortByPriceAndNonce = function() {
   var sortedByNonce = {};
 
   self.pending_transactions.forEach((tx) => {
-    const from = to.hex(tx.from);
-    if (sortedByNonce[from]) {
-      sortedByNonce[from].push(tx);
+    const from = tx.from.toString("hex");
```

Comment:
Thoughts on generating hex values on tx instantiation and held as member variables instead of recalculating `value.toString("hex")` each time?